.gitignore

```diff
@@ -22,4 +22,5 @@
 /web/config.php
 logs/
 build/
+/src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/sftp-config.json
```

Comment:
this shouldnt be here, if you want to ignore it, you need to add it to your local global gitignore file not to the repository